pkg/core/core_test.go

```diff
@@ -45,23 +61,45 @@ func TestEarlybirdCfg_GitClone(t *testing.T) {
 		t.Skip("If test cases not running locally, skip cloning external repositories for CI/CD purposes.")
 	}
 
-	giturl := os.Getenv("giturl")
-	if giturl == "" {
-		t.Skip("Skipping GitClone. Git repository URL is required. Include ENV var: giturl")
-	}
-
 	var (
+		FakeRepo = "https://github.com/carnal0wnage/fake_commited_secrets"
 		RepoUser string
 		Project  string
 	)
 	ptr := PTRGitConfig{
-		Repo:     &giturl,
+		Repo:     &FakeRepo,
 		RepoUser: &RepoUser,
 		Project:  &Project,
 	}
 
 	eb.GitClone(ptr)
 
 	//Delete temporary cloned repository directory
-	utils.DeleteGit(giturl, eb.Config.SearchDir)
+	utils.DeleteGit(FakeRepo, eb.Config.SearchDir)
+}
+
+func TestEarlybirdCfg_getDefaultModuleSettings(t *testing.T) {
+	modules := map[string]cfgReader.ModuleConfig{
+		"inclusivity": {
+			DisplaySeverity: "high",
+		},
+		"rtfUrl": {
```

Comment:
shouldnot be here

Reply:
I'm going to remove - I don't think we want to open source rtf stuff